doc/classes/OS.xml

```diff
@@ -360,6 +360,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Returns the amount of time in milliseconds it took for the boot logo to appear.
```

Comment:
TIL about this method :O
I don't really see why this information is relevant enough to expose in the first place, but the docs are good :)

Reply:
Understood.

Reply:
To be clear, I did not mean this for you, this is just me wondering why we have this method in the first place.